README.md

```diff
@@ -45,48 +46,40 @@ Releases can be found on the [release list][releases].
 
 
 For semi-official builds and third-party packages please see
-[mpv.io](http://mpv.io/installation/).
+https://mpv.io/installation/.
```

Comment:
No.

Reply:
When I see [mpv.io] I expect it to link to https://mpv.io/.